.ci/pipeline_definitions

```diff
@@ -15,7 +15,12 @@ delivery-dashboard:
 
   jobs:
     manual-release:
-      <<: *check-steps
```

Comment:
I think we should also keep the usual lint step in the release job, since the sast one does only contain sast-related rules 🤔